app/code/Magento/CustomerGraphQl/etc/graphql/di.xml

```diff
@@ -20,4 +20,13 @@
             </argument>
         </arguments>
     </type>
+    <type name="Magento\StoreGraphQl\Model\Resolver\Store\StoreConfigDataProvider">
```

Comment:
Do we still need this injection?  As far as I see, the validation process happens in  `\Magento\Customer\Model\AccountManagement::resetPassword` now. But I may miss something. Could you double-check, please?

Reply:
Hi @rogyar .
If  I am not mistake, It doesn't use for validation process. This injection need for adding configs to 
`type Query {
    storeConfig : StoreConfig @resolver(class: "Magento\\StoreGraphQl\\Model\\Resolver\\StoreConfigResolver") @doc(description: "The store config query") @cache(cacheable: false)
}`
Also it was mentioned in https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/customer-password-reset.md  with link on example.

Thank you !

Reply:
That's right, I forgot about whitelisting the config paths. Thank you for the clarification.